query_test.go

```diff
@@ -0,0 +1,90 @@
+package surrealdb
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/surrealdb/surrealdb.go/pkg/marshal"
+)
+
+// TestSurrealDBLocal runs a local SurrealDB instance by calling
+// `NewTestSurrealDB(t)`, which is a dedicated SurrealDB instance with no data.
+func TestSurrealDBLocal(t *testing.T) {
+	t.Parallel()
+	cases := map[string]struct {
+		schema              []string
+		dbInteraction       func(*testing.T, *DB)
+		keepServerUponError bool
+	}{
+		"first case": {
```

Comment:
This was meant to be so that each test case to use `CREATE` keyword, which cannot run against the same table in parallel. Probably I should have named this test func so that it is clearly about table tests can run in parallel -- I had no intention of adding too many test cases in this particular test func.